README.md

```diff
@@ -0,0 +1,243 @@
+# Sawtooth RFCs
+[Sawtooth RFCs]: #sawtooth-rfcs
+
+Many changes, including bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the Sawtooth community and
+the [sub-team]s.
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for major changes to enter Sawtooth Core and other official
+project components, so that all stakeholders can be confident about the
+direction Sawtooth is evolving in.
+
+This process is intended to be substantially similar to the Rust RFCs process,
+customized as necessary for use with Sawtooth. The README.md and
+0000-template.md were initially forked from [Rust
+RFCs](https://github.com/rust-lang/rfcs).
+
+
+## Table of Contents
+[Table of Contents]: #table-of-contents
+
+  - [Opening](#sawtooth-rfcs)
+  - [Table of Contents]
+  - [When you need to follow this process]
+  - [Before creating an RFC]
+  - [What the process is]
+  - [The RFC life-cycle]
+  - [Reviewing RFCs]
+  - [Implementing an RFC]
+  - [Help this is all too informal!]
+  - [License]
+
+
+## When you need to follow this process
+[When you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to
+Sawtooth or any of its sub-components including but not limited to Sawtooth
+Core, Sawtooth Supply Chain, Sawtooth Seth, the various Sawtooth SDKs, or the
+RFC process itself. What constitutes a "substantial" change is evolving based
+on community norms and varies depending
+on what part of the ecosystem you are proposing to change, but may include the
+following.
+
+  - Architectural changes
+  - Substantial changes to component interfaces
+  - New core features
+  - Backward incompatible changes
+
```

Comment:
Consider adding:
- Changes that affect the security of communications or administration

Reply:
Added